src/common/types/store-data/unified-data-interface.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// this is similar to `TestEngine` interface from axe-core
+export interface ScanEngine {
+    name: string;
+    version: string;
+}
+
+export interface OSData {
+    name: string;
+    osVersion?: string;
+    userAgent?: string;
```

Comment:
What type of data is userAgent meant to indicate? Is it the browser? What are the possible multiple meanings?

Reply:
`userAgent` is a string property that contains both the browser name and version. We use this in [AI Web navigator utils](https://github.com/microsoft/accessibility-insights-web/blob/master/src/common/navigator-utils.ts) to get the browser name and version. Technically, I think we could get the name and version from the `userAgent` and populate those properties instead, thus removing the need to store the whole `userAgent`.

Reply:
Just as an example: `userAgent` for my current browser is

`"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/76.0.3809.100 Safari/537.36"`